index.html

```diff
@@ -22,9 +22,10 @@
     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
       <div class="navbar-nav">
         <a class="nav-item nav-link active" id="log-in-nav" href="#">Log In</a>
+        <a class="nav-item nav-link hidden" id="log-out-nav" href="#">Log Out</a>
```

Comment:
"Log in" remains while a new user is logged in. No, log out link. No login or log out link if an existing user logs in.

Reply:
My fault i was one commit behind on your branch